cypress-tests/cypress/e2e/spec/Payment/00031-UCSFlowTest.cy.js

```diff
@@ -0,0 +1,184 @@
+import * as fixtures from "../../../fixtures/imports";
+import State from "../../../utils/State";
+import { payment_methods_enabled } from "../../configs/Payment/Commons";
+import getConnectorDetails, * as utils from "../../configs/Payment/Utils";
+
+let globalState;
+
+describe("UCS Flow Testing", () => {
+  const UCS_SUPPORTED_CONNECTORS = ["authorizedotnet"];
+  const UCS_REQUEST_NAMES = [
+    "UCSZeroAuthMandate",
+    "UCSConfirmMandate",
+    "UCSRecurringPayment",
+    "No3DSAutoCapture",
+    "No3DSManualCapture",
+  ];
+
+  before("seed global state", () => {
+    cy.task("getGlobalState").then((state) => {
+      globalState = new State(state);
+    });
+  });
+
+  after("flush global state", () => {
+    cy.task("setGlobalState", globalState.data);
+  });
+
+  const currentConnector = Cypress.env("CYPRESS_CONNECTOR");
+  const isUCSSupported = UCS_SUPPORTED_CONNECTORS.includes(currentConnector);
```

Comment:
can you please follow the cypress conventions that are followed across the previous 30 spec files? you do not need to do all these things.